Automattic/jetpack
# Writing: Copy a post.

This PR adds the Copy Post feature, already available on WordPress.com (as "Duplicate" in the post edit More menu). It is added as a searchable module (not autoactivated), part of the main Writing feature.

#### Changes proposed in this Pull Request:

A new `Copy` item is added to row actions in the post list view. Clicking the link redirects to `/wp-admin/post-new.php?jetpack-copy=postId`, and loads the new draft post with the following properties of the source post:  **title, content, except, featured image, post type, post format, categories, and tags**.

<img width="396" alt="screen shot 2019-01-04 at 1 14 55 pm" src="https://user-images.githubusercontent.com/349751/50861782-96ce7380-134e-11e9-95aa-ed50700a411e.png">

#### Testing instructions:





* Activate the module by searching for it in `/wp-admin/admin.php?page=jetpack#/settings`.
* Create a test post that has the following: title, content, except, featured image, post format, categories, and tags (be sure the theme supports Post Formats, like Twenty Seventeen; Twenty Nineteen does not).
* From `/wp-admin/edit.php`, hover over the test post and then click `Copy`.
* Verify the draft post that loads contains all of the data from the existing post.
* Publish, and verify all information was saved without errors.
* Repeat the same with pages and a custom post type.

#### Proposed changelog entry for your changes:


* new feature: Copy Post. Add a new post based on an existing post.
